### DISCUSSION TITLE
Search in message value

### DISCUSSION THREAD

**Discussion by DianaSensei** [tchiotludo/akhq#695] [Q&A]
Hello everyone, I installed akhq and connected to my kafka. I need to search data in message value, but I don't know how it work. I guess that it use `or` operation. Can anyone help me?

**Comment by tchiotludo** [comment]
You need to browse the topic data here : 
![image](https://user-images.githubusercontent.com/2064609/118376893-fc8c3380-b5ca-11eb-9add-442eaa5ffd50.png)

And after you fill the form ? 
Not really sure to understand your issue :thinking:

**Reply by DianaSensei** [reply]
Yes, I know this form. I have a message A with data 
```
{
  "requestId": "1234567890000",
  "data": "abcd"
}
```
the value of field `requestId` is always unique.
However, when i pass a data in to `Value`,  i meet 2 case:
- Case 1: ex: `1234567890000` -> result message A (ok)
- Case 2: ex: `"requestId": "1234567890000"` -> result not match with message A?

**Reply by tchiotludo** [reply]
How is serialized your data in Kafka ? 
Avro, json ?
